src/web/timeline.rkt

```diff
@@ -178,6 +179,32 @@
 (define (average . values)
   (/ (apply + values) (length values)))
 
+(define (render-phase-mixed-sampling mixsample)
+  `((dt "Precisions")
+    (dd (details
+         (summary "Click to see histograms")
+         ,@(for/list ([rec (in-list (group-by first mixsample))])
+             
+             (define n (random 100000))
+             (define op (car (car rec)))
+             (define precs-times-total (foldr (lambda (x l)
+                                                (list (cons (second x) (first l))
+                                                      (cons (third x) (second l))
+                                                      (+ (third x) (third l))))
+                                              (list '() '() 0)
+                                              rec))
```

Comment:
This is pretty ugly, I think you're trying to avoid looping multiple times. Why not just do:
```
(define precisions (map second rec))
(define times (map third rec))
(define total-time (apply + times))
```

Reply:
Yeah, I also think so, changed to maps